CHANGELOG.md

```diff
@@ -2,6 +2,506 @@
+
+### Allow expressions in more locations in Connectors URIs ([PR #7220](https://github.com/apollographql/router/pull/7220))
+
+Previously, we only allowed expressions in very specific locations in Connectors URIs:
+
+1. A path segment, like `/users/{.id}`
+2. A query parameter's _value_, like `/users?id={.id}`
+
+Expressions can now be used anywhere in or after the path of the URI.
+For example, you can do
+`@connect(http: {GET: "/users?{.filterName}={.filterValue}"})`.
+The result of any expression will _always_ be percent encoded.
+
+> Note: Parts of this feature are only available when composing with Apollo Federation v2.11 or above (currently in preview).
+
+By [@dylan-apollo](https://github.com/dylan-apollo) in https://github.com/apollographql/router/pull/7220
+
+### Enables reporting of persisted query usage by PQ ID to Apollo ([PR #7166](https://github.com/apollographql/router/pull/7166))
+
+This change allows the router to report usage metrics by persisted query ID to Apollo, so that we can show usage stats for PQs.
+
+By [@bonnici](https://github.com/bonnici) in https://github.com/apollographql/router/pull/7166
+
+### Instrument coprocessor request with http_request span ([Issue #6739](https://github.com/apollographql/router/issues/6739))
+
+Coprocessor requests will now emit an `http_request` span. This span can help to gain
+insight into latency that may be introduced over the network stack when communicating with coprocessor.
+
+Coprocessor span attributes are:
+* `otel.kind`: `CLIENT`
+* `http.request.method`: `POST`
+* `server.address`: `<target address>`
+* `server.port`: `<target port>`
+* `url.full`: `<url.full>`
+* `otel.name`: `<method> <url.full>`
+* `otel.original_name`: `http_request`
+
+By [Jon Christiansen](https://github.com/theJC) in https://github.com/apollographql/router/pull/6776
+
+### Enables reporting for client libraries that send the library name and version information in operation requests. ([PR #7264](https://github.com/apollographql/router/pull/7264))
+
+Apollo client libraries can send the library name and version information in the `extensions` key of an operation request. If those values are found in a request the router will include them in the telemetry operation report.
+
+By [@calvincestari](https://github.com/calvincestari) in https://github.com/apollographql/router/pull/7264
+
+### Add compute job pool spans ([PR #7236](https://github.com/apollographql/router/pull/7236))
+
+The compute job pool in the router is used to execute CPU intensive work outside of the main I/O worker threads, including GraphQL parsing, query planning, and introspection.
+This PR adds spans to jobs that are on this pool to allow users to see when latency is introduced due to
+resource contention within the compute job pool.
+
+* `compute_job`:
+  - `job.type`: (`QueryParsing`|`QueryParsing`|`Introspection`)
+* `compute_job.execution`
+  - `job.age`: `P1`-`P8`
+  - `job.type`: (`QueryParsing`|`QueryParsing`|`Introspection`)
```

Comment:
These were snake-case, rather than camelcase, in the 1.x release line.  Is that intentionally different?

Reply:
No, these should be snake case. It also an edit we had to do for the 1.61.5 release we just did.

Reply:
fixed.